src/Pos/Client/Txp/Util.hs

```diff
@@ -111,15 +121,15 @@ makeRedemptionTx rsk txInputs = makeAbstractTx mkWit (map ((), ) txInputs)
             }
 
 type FlatUtxo = [(TxIn, TxOutAux)]
-type InputPicker = StateT (Coin, FlatUtxo) (Either TxError)
+type InputPicker = StateT (Coin, FlatUtxo) (Either Text)
 
 -- | Given Utxo, desired source addresses and desired outputs, prepare lists
 -- of correct inputs and outputs to form a transaction
 prepareInpsOuts
     :: Utxo
     -> NonEmpty Address
     -> TxOutputs
-    -> Either TxError (TxOwnedInputs Address, TxOutputs)
+    -> Either Text (TxOwnedInputs Address, TxOutputs)
```

Comment:
I think it makes more sense to return `TxError` from this function if something goes wrong.